lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -292,31 +383,93 @@ class MockTimers {
     }
   }
 
-  enable(timers = SUPPORTED_TIMERS) {
+  enable(...args) {
     if (this.#isEnabled) {
-      throw new ERR_INVALID_STATE(
-        'MockTimers is already enabled!',
-      );
+      throw new ERR_INVALID_STATE('MockTimers is already enabled!');
+    }
+
+    if (args.length <= 0) {
```

Comment:
if (!args.length)  as args cannot be a negative number

Reply:
Did this mostly to be opposed to `args > 0`, the intent was to have it as  `args === 0`, will change

Reply:
This will not be needed when it's an option object

Reply:
still nicer to be explicit instead of relying on the falsiness of zero